update reputations command - is array 1 (#15145)
demisto/content: Packs/CVESearch/ReleaseNotes/1_0_9.md

#### Integrations
##### CVE Search v2

> - You can now pass an array to the ***cve*** command.

Reviewer: ```suggestion
- Added array input support for the ***cve*** command.
```
